### PR TITLE
feat: wcag 2.1 aa compliance update #1916

### PR DESCRIPTION
**Pull request by Pushpajyothi**
Issue : https://github.com/awslabs/iot-app-kit/issues/1916

Description :
Added @axe-core/playwright package to create automation test suites for accessibility.
Created automation test suites to check WCAG 2.1 AA Compliance Accessibility issues.
Playwright test cmd will also run accessibility tests.(yarn run test:e2e)

Note : For now we have skip the testcases so that PR can merge.

### CODE REVIEW

**Comment by tracy-french** [comment]
@Pushpajyothi Linting is failing. Could you take a look at that?

File: tests/accessibility/accessibility.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { expect, Page } from '@playwright/test';
+import AxeBuilder from '@axe-core/playwright';
+
+export async function accessibilityTest(page: Page) {
```
Comment (tracy-french) [inline tests/accessibility/accessibility.ts:4]: Accessibility testing! Nice!

File: tests/accessibility/create-dashboard.spec.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { test, expect } from '@playwright/test';
+import { CreateDashboardPage } from '../pages/create-dashboard.page';
+import { accessibilityTest } from './accessibility';
+
+test.skip(' Accessibility > As a user, I can verify Create Dashboard Page ', async ({
```
Comment (tracy-french) [inline tests/accessibility/create-dashboard.spec.ts:5]: When do we stop skipping the tests?

**Review by tracy-french — CHANGES_REQUESTED** [review]

File: tests/pages/dashboards-index.page.ts [hunk 3]
```diff
@@ -158,6 +162,10 @@ export class DashboardsIndexPage {
     this.dashboardFilter = page.getByRole('form', {
       name: 'Filter dashboards',
     });
+    //Added View Button
```
Comment (dpportet) [inline tests/pages/dashboards-index.page.ts:165]: If you run `yarn lint:fix` it should fix a lot of spacing issues like adding a space after the comment `//`.

**Review by dpportet — CHANGES_REQUESTED** [review]

**Review by mnischay — CHANGES_REQUESTED** [review]
Nice, i was under the impression that all the accessibility test are ran as a different step in GH actions ... i dont see those changes.

**Comment by Pushpajyothi** [comment]
Due to conflicts closed this PR and raised new PR in IoT application

**Pull request closed without merging**